test/copyPaste.test.tsx

```diff
@@ -81,117 +81,117 @@ function setup(onPasteCallback = true, onCopyCallback = false) {
 test('should not allow copy/paste if onPaste & onCopy is undefined', async () => {
   setup(false, false);
   await userEvent.click(getCellsAtRowIndex(0)[0]);
-  copySelectedCell();
+  await copySelectedCell();
   expect(getSelectedCell()).not.toHaveClass(copyCellClassName);
   expect(onCopySpy).not.toHaveBeenCalled();
-  await userEvent.keyboard('{arrowdown}');
-  pasteSelectedCell();
+  await userEvent.keyboard('{enter}{arrowdown}');
```

Comment:
`Enter` closes the editor

Reply:
Why is the editor open at this point?

Reply:
It is actually the [paste operation](https://github.com/adazzle/react-data-grid/pull/3431/files#diff-fa609423a6b1af12decc8ac09d765500df528293166210e9220eb9bfd39bb5aaR57) that triggers the editor. Also, `escape` does not work with `null` editor